libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -72,6 +72,15 @@ class YulUtilFunctions
 	/// Pads with zeros and might write more than exactly length.
 	std::string copyToMemoryFunction(bool _fromCalldata);
 
+	/// @returns a function name that copies string literal to memory
```

Comment:
```suggestion
	/// @returns the name of a function that copies a string literal to memory
```
I think it's clearer that way, like the other docs